javascript/javascript2/week3/game.js

```diff
@@ -0,0 +1,87 @@
+const button = document.querySelector("#button-start-game");
+const input = document.querySelector("#input-time");
+const boxWinner = document.querySelector("#box-winner");
+const showCounterS = document.querySelector("#counter-s");
+const showCounterL = document.querySelector("#counter-l");
+
+let confettiSettingsS = { target: "canvas-s" };
+let confettiS = new ConfettiGenerator(confettiSettingsS);
+
+let confettiSettingsL = { target: "canvas-l" };
+let confettiL = new ConfettiGenerator(confettiSettingsL);
+
+let countS = 0;
+let countL = 0;
+let gameStarted = false;
+
+//What if a user starts pressing a key before the game is started, what should happen?
+
+window.addEventListener("keydown", showMessage);
+
+function showMessage(event) {
+  if (event.key === "s" && gameStarted === false) {
+    alert("enter time to start a game");
+  } else if (event.key === "l" && gameStarted === false) {
+    alert("enter time to start a game");
+  }
+}
+
+function countKeyPressed(event) {
+  if (event.key) {
+    if (event.key === "s") {
+      countS++;
+
+      showCounterS.innerText = countS;
+    } else if (event.key === "l") {
+      countL++;
+
+      showCounterL.innerText = countL;
+    }
+  }
+}
+function startGame() {
+  //What if no time was specified for the game?
+  if (input.value === "") {
+    alert("Enter time to start");
+    throw new Error("Enter time to start");
+  }
+  //If input is not a number
+  if (isNaN(input.value)) {
+    alert("Time should be in seconds - number");
+    throw new Error("Time should be in seconds - number");
+  }
```

Comment:
Putting errors in the code is very smart! It makes it very user friendly and helps debugging the code easier! very good practice!